sevntu-checks/src/test/java/com/github/sevntu/checkstyle/checks/coding/RequireFailForTryCatchInJunitCheckTest.java

```diff
@@ -116,6 +116,14 @@ public void testJunit4And5Mix() throws Exception {
         verify(checkConfig, getPath("InputRequireFailForTryCatchInJunitCheck7.java"), expected);
     }
 
+    @Test
+    public void testStaticWildCardImport() throws Exception {
+        final DefaultConfiguration checkConfig =
+                createModuleConfig(RequireFailForTryCatchInJunitCheck.class);
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
```

Comment:
Sorry, 1 minor change. Please add an example showing a violation where fail is missing using your wildcard import.

Reply:
Done.